src/components/widgets/console/ConsoleCard.vue

```diff
@@ -21,6 +21,24 @@
     </template>
 
     <template v-slot:menu>
+
+      <v-tooltip
+        v-if="scrollingPaused"
+        top>
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn
+            @click="console.scrollToBottom(true)"
+            v-bind="attrs"
+            v-on="on"
+            fab
+            small
+            text>
+            <v-icon>$down</v-icon>
+          </v-btn>
+        </template>
+        <span>{{ $t('app.console.placeholder.scroll') }}</span>
+      </v-tooltip>
```

Comment:
From my understanding, the `app-btn` is the preferred way to get this button here:

```suggestion

      <app-btn
        v-if="scrollingPaused"
        @click="console.scrollToBottom(true)"
        color=""
        fab x-small text>
        <v-icon>$down</v-icon>
      </app-btn>
```